public/locales/en/pageNotFound.json

```diff
@@ -0,0 +1,4 @@
+{
+  "heading": "Page Not Found",
+  "message": "The page you are looking for was not found. If you believe this is an error, please contact Demo Dataverse Support for assistance."
```

Comment:
This is fine for now, I suppose, but in the future, instead of "Demo Dataverse Support" as a hard-coded string, we'll want to pull this from the backend.

Reply:
Yes, I saw there is a Modal to contact Dataverse Support, we can create a new issue to add this feature